addons/survey/models/survey_user_input.py

```diff
@@ -19,6 +19,7 @@ class SurveyUserInput(models.Model):
     _description = "Survey User Input"
     _rec_name = "survey_id"
     _order = "create_date desc"
+    _inherit = ['mail.thread', 'mail.activity.mixin']
```

Comment:
Note that commit message does not seem to explain why we need this (especially activities, seem a bit overkill to me)